src/mon/FSCommands.cc

```diff
@@ -441,6 +441,21 @@ class SetHandler : public FileSystemCommandHandler
       {
         fs->mds_map.set_max_filesize(n);
       });
+    } else if (var == "max_xattr_size") {
+      if (interr.length()) {
+	ss << var << " requires an integer value";
+	return -EINVAL;
+      }
+      if (n < MDS_MAX_XATTR_SIZE) {
+	ss << var << " must at least " << MDS_MAX_XATTR_SIZE;
+	return -ERANGE;
```

Comment:
Why fail it here ? Shouldn't the `MDS_MAX_XATTR_SIZE` just a default value ? Users should be allowed to set any value larger or less than it, right ?

Reply:
Sure, I can drop this check.  I'm not really sure if in which situations admins should be really playing with this setting anyway.